tests/models/whisper/test_processor_whisper.py

```diff
@@ -177,3 +178,296 @@ def _test_prompt_error_raised_helper(prompt, special_token):
         _test_prompt_error_raised_helper("<|startofprev|> test", "<|startofprev|>")
         _test_prompt_error_raised_helper("test <|notimestamps|>", "<|notimestamps|>")
         _test_prompt_error_raised_helper("test <|zh|> test <|transcribe|>", "<|zh|>")
+
+    def test_find_longest_common_subsequence_old(self):
```

Comment:
moved from the ASR pipeline tests -- it is a processor test, not a pipeline test